packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Link/index.js

```diff
@@ -30,12 +30,22 @@ class LinkContainer extends Component {
       if (e.data.command === 'enablePlaid') this.setState({ enablePlaid: true })
       if (e.data.command === 'disablePlaid') this.setState({ enablePlaid: false })
       if (e.data.command === 'getBankAccounts' && e.data.msg) {
+        this.props.sfoxFrontendActions.sfoxLoading()
```

Comment:
Looking at the reducer I see all frontend actions sfoxLoading, sfoxNotAsked, sfoxFailure and sfoxSuccess are actually reduced in sfoxBusy so the naming is very misleading. You should rename them sfoxBusyLoading and so on

Reply:
Ahh, that's a good point 👍